doc/user_guide/user_guide.md

```diff
@@ -0,0 +1,291 @@
+  "host": "<SAAS_HOST>",
+  "account_id": "<SAAS_ACCOUNT_ID>",
+  "database_id": "<SAAS_DATABASE_ID>",
+  "pat": "<SAAS_PAT>"
+  }'
+USER '{"username": "<USER_NAME>"}'
+IDENTIFIED BY '{"password": "<PASSWORD>"}' ;
+```
+
+The list of elements in the connection's parameter called `TO` depends on the
+backend you want to use. There are two different backends: `onprem` and
+`saas`.
+
+The following table shows all elements for each of the backends.
+
+| Backend  | Parameter            | Required? | Default value  | Description                                                        |
+|----------|----------------------|-----------|----------------|--------------------------------------------------------------------|
+| (any)    | `<CONNECTION_NAME>`  | yes       | -              | Name of the connection                                             |
+| (any)    | `<USER_NAME>`        | -         | `true`         | Name of the user accessing the Bucket (requires  write permissions) |
+| (any)    | `<PASSWORD>`         | -         | `true`         | Password for accessing the Bucket (requires  write permissions)    |
+| (any)    | `<BACKEND>`          | yes       | -              | Which backend to use, must be either `onprem` or `saas`            |
+| `onprem` | `<HOST>`             | yes       | -              | Fully qualified Hostname or ip Address                             |
+| `onprem` | `<PORT>`             | -         | `2580`         | Port of the BucketFS Service                                       |
+| `onprem` | `<SERVICE_NAME>`     | yes       | `bfsdefault`   | Name of the BucketFS Service                                       |
+| `onprem` | `<BUCKET_NAME>`      | yes       | `default`      | Name of the Bucket                                                 |
+| `onprem` | `<PATH>`             | -         | (empty / root) | Path inside the Bucket                                             |
+| `onprem` | `<VERIFY>`           | -         | `true`         | Whether to apply TLS security to the connection                    |
+| `saas`   | `<SAAS_ACCOUNT_ID>`  | yes       | -              | Account ID for accessing an SaaS database instance                 |
+| `saas`   | `<SAAS_DATABASE_ID>` | yes       | -              | Database ID of an Exasol SaaS database instance                    |
+| `saas`   | `<SAAS_PAT>`         | yes       | -              | Personal access token for accessing an SaaS database instance      |
+
+### AAF Python Package
+
+The latest version of AAF can be obtained from [pypi](https://pypi.org), see also the [releases on GitHub](https://github.com/exasol/advanced-analytics-framework/releases).
+
+The following command installs the AAF from pypi:
+
+```bash
+pip install exasol-advanced-analytics-framework
+```
+
+### Script Language Containers (SLCs)
+
+Exasol executes User Defined Functions (UDFs) in an isolated Script Language Container (SLCs).
+Running the AAF requires a custom SLC.
+
+#### Build the SLC
```

Comment:
I think, this section we can move to the developer guide, because we will release the prebuilt slc

Reply:
See next push